docs/CONTRIBUTING.md

```diff
@@ -47,7 +42,31 @@ Get your own API key:
 1. Set your redirect url to `https://127.0.0.1:8080/return.html` (or whatever the IP or hostname is of your dev server)
 1. Select all scopes _except_ the Administrate Groups/Clans
 1. Enter `https://127.0.0.1:8080` as the `Origin Header`
-1. Run `yarn install && yarn start`
+
+### Start Dev Server
+
+* Run `yarn install`
+* Run `yarn start`
+
+*Note:* on Windows, when running `yarn install` the Git Bash shell may fail to fetch all necessary packages even when run as Admin ([details](https://github.com/DestinyItemManager/DIM/issues/2487)). If that's the case, simply use cmd as Admin instead.
+
+### Enter API Credentials
+
+This step will need to be done each time you clear your browser cache. You will be automatically redirected to a screen to enter these credentials
+if the app can't load them from local storage when it starts.
+
+1. Open your browser and navigate to [https://localhost:8080]()
 1. Copy your API-key, Oauth Client_id, and OAuth client_secret from bungie.net into DIM developer settings panel when it is loaded.
 
-Check out the [docs](https://github.com/DestinyItemManager/DIM/blob/master/docs) folder for more tips.
+### Development
+
+**Overview**
+
+The `yarn start` step will create a hot-loading webserver, and a TLS cert/key pair. You will access your local development site by visiting [https://localhost:8080]().
+You will likely get a security warning about the certificate not being trusted. This is because it's a self-signed cert generated dynamically for your environment,
+and is not signed by a recognized autority. Dismass/advance past these warning to view your local DIM application.
+
+**Check code Style**
+
+* `yarn lint` will tell you if you're following the DIM code style (and automatically fix what it can).
+Check out the [docs]() folder for more tips.
```

Comment:
Link?

Reply:
That's actually the right link! IKR!? I had to force commit 3 times so I didn't look like an idiot with my commits trying to get that one to work.